react/features/mobile/callkit/middleware.js

```diff
@@ -228,14 +228,17 @@ function _conferenceWillJoin({ getState }, next, action) {
     const { conference } = action;
     const state = getState();
     const { callUUID } = state['features/base/config'];
+    const { callHandle } = state['features/base/config'];
```

Comment:
That can be shortened into `const { callHandle, callUUID } = state['features/base/config'];`